server/admin_management/api/app/routers/router_guards.py

```diff
@@ -83,6 +86,26 @@ async def get_current_requester(
     return requester
 
 
+# for delegated admin, there is no access group in the token, our auth lambda function only add app admin to the token
```

Comment:
I thought we were adding a generic DELEGATED_ADMIN group to the token for delegated admins

Reply:
Hi Basil! We don't add "DELEGATED_ADMIN" to the token currently, and I feel it's okay. Because when we check if the user can make the api call or not, he needs to be the delegated admin of the selected application, with just "DELEGATED_ADMIN" in the token is not enough. For app admin, we do show he's the admin of which application ("FOM_DEV_ADMIN", "FAM_ADMIN"), so that's fine to check the token. But for delegated admin, we can't just use the "DELEGATED_ADMIN" anyway, need to check the real access.

Reply:
I know we were talking about it before, so I just wanted to make sure. What you've done above seems fine.

Reply:
thanks!! So for the admin management apis, they all need app admin role, just that get my access api doesn't need to check role, because the purpose of that api is to return the login user's access. 

For the apis under our server/backend, which will be used by delegated admin/app admin to do user access management, we need to adjust the code to check the real access. We have ticket for that already in backlog. Just want to mention it here as a note.